apps/commudle-admin/src/app/feature-modules/hackathon-control-panel/components/hackathon-control-panel-mentors/hackathon-control-panel-mentors.component.html

```diff
@@ -121,6 +120,24 @@
   </nb-card>
 </ng-template>
 
+<ng-template #promoteTeams let-data let-ref="dialogRef">
+  <nb-card>
+    <nb-card-header>Promote Teams to Next Round</nb-card-header>
+    <nb-card-body>
+      <p>
+        This will promote teams and their evaluator assignments to <strong>{{ data.nextRoundName }}</strong
```

Comment:
This will allocate the same mentors to teams which have been promoted to <bold>roundname</bold>